packages/rum-core/src/domain/configuration.ts

```diff
@@ -39,6 +39,7 @@ export interface RumInitConfiguration extends InitConfiguration {
   startSessionReplayRecordingManually?: boolean | undefined
 
   // action options
+  enablePrivacyForActionName?: boolean | undefined
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Could be nice to add a comment saying `// TODO next major: remove this option and make privacy for action name the default behaviour`